src/style.css

```diff
@@ -0,0 +1,147 @@
+html,
+body,
+div,
+a,
+ul,
+p,
+li,
+header,
+nav {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --mane-text-color: hsl(234, 80%, 12%);
+  --secondary-text-color: hsl(244, 8%, 41%);
+  --button-color: hsl(193, 100%, 43%);
+}
+
+.Card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
```

Comment:
do not hardcode height. content of card should make this height
![Screenshot from 2023-06-10 17-10-59](https://github.com/mate-academy/layout_product-cards/assets/33029864/553c4c78-bc1b-4288-abba-0e5076f60b68)

Reply:
Done